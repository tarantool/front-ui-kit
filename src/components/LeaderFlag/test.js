import React from 'react';
import renderer from 'react-test-renderer';
import { LeaderFlag } from './index';

it('LeaderFlag renders correctly', () => {
  const tree = renderer.create(
    <>
      <LeaderFlag state='good' title='Good' />
      <LeaderFlag state='bad' title='Bad' />
      <LeaderFlag state='warning' title='Bad' />
    </>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
