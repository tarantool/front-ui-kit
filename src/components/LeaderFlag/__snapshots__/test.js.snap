// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeaderFlag renders correctly 1`] = `
Array [
  .emotion-0 {
  position: relative;
  width: 14px;
  height: 17px;
  overflow: hidden;
  -webkit-transition: height 0.3s ease-in-out;
  transition: height 0.3s ease-in-out;
}

.emotion-0:hover {
  height: 59px;
}

.emotion-1 {
  width: 14px;
  height: 59px;
  position: absolute;
  left: 0;
  bottom: 0;
  fill: #b5ec8e;
}

<div
    className="emotion-0"
    title="Good"
  >
    <svg
      className="emotion-1"
      viewBox="0 0 14 59"
    >
      <use
        xlinkHref="#flag"
      />
    </svg>
  </div>,
  .emotion-0 {
  position: relative;
  width: 14px;
  height: 17px;
  overflow: hidden;
  -webkit-transition: height 0.3s ease-in-out;
  transition: height 0.3s ease-in-out;
}

.emotion-0:hover {
  height: 59px;
}

.emotion-1 {
  width: 14px;
  height: 59px;
  position: absolute;
  left: 0;
  bottom: 0;
  fill: #ff272c;
}

<div
    className="emotion-0"
    title="Bad"
  >
    <svg
      className="emotion-1"
      viewBox="0 0 14 59"
    >
      <use
        xlinkHref="#flag-light"
      />
    </svg>
  </div>,
  .emotion-0 {
  position: relative;
  width: 14px;
  height: 17px;
  overflow: hidden;
  -webkit-transition: height 0.3s ease-in-out;
  transition: height 0.3s ease-in-out;
}

.emotion-0:hover {
  height: 59px;
}

.emotion-1 {
  width: 14px;
  height: 59px;
  position: absolute;
  left: 0;
  bottom: 0;
  fill: #faad14;
}

<div
    className="emotion-0"
    title="Bad"
  >
    <svg
      className="emotion-1"
      viewBox="0 0 14 59"
    >
      <use
        xlinkHref="#flag"
      />
    </svg>
  </div>,
]
`;
